org.libredirect.frontends-manager.yml

```diff
@@ -0,0 +1,52 @@
+id: org.libredirect.frontends-manager
+runtime: org.gnome.Platform
+runtime-version: '42'
```

Comment:
this runtime has been EOL for a year.

Reply:
Bc tauri still doesn't support newer webgtk.

Reply:
> Bc tauri still doesn't support newer webgtk.

Are you able to use 2.0 alpha builds in the project?

Reply:
I'll try to upgrade the gnome platform version as soon as 2.0 gets stable.